src/lib.rs

```diff
@@ -333,6 +332,7 @@ impl Error {
     }
 }
 
+#[allow(clippy::from_over_into)]
```

Comment:
Generally I don't think that this impl is good and would rather remove it. However, I did not want to break users with this PR.

Reply:
Agreed